validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -79,25 +116,51 @@ def __init__(self, block_cache, state_view, data_dir):
         self._state_view = state_view
 
     def verify_block(self, block_state):
```

Comment:
Small nit.  The verify_block method really takes a BlockWrapper object, so even though it is just a variable name, block_wrapper would more accurately convey its use/meaning.